{{cookiecutter.project_dir}}/.lighthouse/jenkins-x/python/release.yaml

```diff
@@ -61,7 +62,7 @@ spec:
                 #!/busybox/sh
                 source .jx/variables.sh
                 cp /tekton/creds-secrets/tekton-container-registry-auth/.dockerconfigjson /kaniko/.docker/config.json
-                /kaniko/executor $KANIKO_FLAGS --context=/workspace/source/{{cookiecutter.module_name}}_api --dockerfile=${DOCKERFILE_PATH:-Dockerfile} --destination=$PUSH_CONTAINER_REGISTRY/$DOCKER_REGISTRY_ORG/$APP_NAME:$VERSION
+                /kaniko/executor $KANIKO_FLAGS --context=/workspace/source --dockerfile=${DOCKERFILE_PATH:-Dockerfile} --destination=$PUSH_CONTAINER_REGISTRY/$DOCKER_REGISTRY_ORG/$APP_NAME:$VERSION
```

Comment:
did we miss the `apps/api` in --context?

Reply:
No @tuan. The context must be set to the top level because we will need to copy `common` folder which contains shared functions to each python-based app. You can check the Dockerfile for more info.